src/lib/isURL.js

```diff
@@ -4,6 +4,18 @@ import isFQDN from './isFQDN';
 import isIP from './isIP';
 import merge from './util/merge';
 
+/*
+options for isURL method
```

Comment:
Glad to see you have added this. 
In my mind, I was thinking that `documentation` was done is something like `README.md` or `wiki` or `website`. 

Or when we talk about `documentation in this case`, we mean documenting for the `developers` who are `contributing`? 

Please let me know @rohankulshreshtha.

Reply:
@ezrqnkemboi Thanks for your input.
I added the comments in `isURL.js` file as a documentation for developers who are contributing.
Now I have also added the same in `README.md` (under the isURL section) for others to read as well.
Please let me know if any other changes are required.